bundles/org.openhab.core/src/main/java/org/openhab/core/common/PoolBasedSequentialScheduledExecutorService.java

```diff
@@ -100,22 +120,24 @@ public PoolBasedSequentialScheduledExecutorService(ScheduledThreadPoolExecutor p
                     tail = empty;
                 }
             }
-        }, 2, 4, TimeUnit.SECONDS);
+        },
+                // avoid cleaners of promptly created instances to run at the same time
```

Comment:
the indentation does not look right here.

Reply:
Weird - I guess this is some bug in the spotless formatter. Well, let's leave it as it is then!